server/repository/file_pairs.go

```diff
@@ -24,10 +24,10 @@ func (repo *FilePairs) getWithQuery(queryRow scannable) (*model.FilePair, error)
 
 	err := queryRow.Scan(&pair.ID,
 		&pair.Left.BlobID, &pair.Left.RepositoryID, &pair.Left.CommitHash,
-		&pair.Left.Path, &pair.Left.Content, &pair.Left.Hash,
+		&pair.Left.Path, &pair.Left.Content, &pair.Left.UAST, &pair.Left.Hash,
```

Comment:
In some cases, UAST can be really big, and it is not needed for printing FilePairs, so it could be removed from the query.
Let's keep it in mind for the future in case we notice that getting the next file pair is delaying too much